packages/rum-core/src/domain/rumEventsCollection/view/trackScrollMetrics.ts

```diff
@@ -0,0 +1,40 @@
+import type { Duration } from '@datadog/browser-core'
+import { throttle, addEventListener, DOM_EVENT } from '@datadog/browser-core'
+
+export interface ScrollMetrics {
+  maxScrollDepth?: number
+  maxscrollHeight?: number
+  maxScrollDepthTimestamp?: Duration
+}
+
+export function trackScrollMetrics() {
+  const scrollMetrics: ScrollMetrics = {}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Shouldn't this object be initialized at the beginning?

Reply:
What do you mean by initialized? Set a default scroll depth, timestamp etc...?

Reply:
Not default, but rather set the current values. What if the document was already scrolled down when the view is created?